src/guides/install-yellow-light-keeps-flashing-in-heartbeat.html

```diff
@@ -24,16 +24,54 @@
     <span>
       <ol>
         <li>Download the Yellow operating system.
-          <ul><li>Go to the <a href="https://github.com/home-assistant/operating-system/releases">operating system page</a> on github and download the latest version of the `haos_yellow-xx.x.img.xz` file.
-          </li></ul></li>
-          <li>Reinstall the operating system on Yellow. <ul><li>Open the procedure on <a href="/guides/reinstall-os/#install-os-rpiboot">how to reinstall the operating system</a>. </li><li>Follow the procedure <i>Option 2: Reinstall Home Assistant OS using rpiboot</i>  up to step 8.</li>
-          <li>In step 9, use the downloaded image instead of the image provided by the Raspberry Pi Imager.</li>
-          <li>Follow the rest of the steps to finish reinstalling the Yellow operating system.</li>
+          <ul><li>Go to the <a href="https://github.com/home-assistant/operating-system/releases">operating system page</a> on github and download the latest stable version of the `haos_yellow-xx.x.img.xz` file.
+          </li></ul>
+        </li>        
+        <li>
+          <span>Install rpiboot onto your computer.</span>
+          <ul>
+            <li>
+              <span
+                >Using
+                <a
+                  href="https://github.com/raspberrypi/usbboot/raw/master/win32/rpiboot_setup.exe"
+                  >Windows Installer</a
+                ></span
+              >
+            </li>
+            <li>
+              <span
+                >From 
+                <a
+                  href="https://www.raspberrypi.com/documentation/computers/compute-module.html#building-rpiboot-on-your-host-system-cygwinlinux"
+                  >source code</a
+                ></span
+              >
+            </li>
+          </ul>
+        </li>
```

Comment:
This step is already in the other procedure, so I'd not duplicate it here.
```suggestion
```

Reply:
I realize there is duplication. But I added it there on purpose because I find it more convenient to download everything first. So that you have it all ready before you start executing the steps.